ipatests/test_integration/test_commands.py

```diff
@@ -1561,6 +1561,54 @@ def test_ipa_cacert_manage_prune(self):
 
         assert isrgrootx1_nick in result
 
+    def test_ipa_systemd_journal(self):
+        """
+        This testcase checks that administrative user credentials
+        is not leaked to journald log when ipa commands are run.
+        """
+        tasks.kinit_admin(self.master)
+        tasks.kinit_admin(self.replicas[0])
+        tasks.kinit_admin(self.clients[0])
+        cmds = [
+            ['ipa-adtrust-install', '-a',
+             self.master.config.admin_password, '-U'],
+            ['ipa-replica-manage', '-p',
+             self.master.config.dirman_password, 'list'],
```

Comment:
This command does not log anything in the journal, see the report.html:
```
INFO     ipatests.pytest_ipa.integration.host.Host.master.IPAOpenSSHTransport:transport.py:391 RUN ['journalctl', '-t', '/usr/sbin/ipa-replica-manage', '-n1', '-o', 'json-pretty']
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd82:transport.py:513 RUN ['journalctl', '-t', '/usr/sbin/ipa-replica-manage', '-n1', '-o', 'json-pretty']
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd82:transport.py:217 Exit code: 0
```
You need to call instead `['ipa-replica-manage', 'del', f"dummyhost.{self.master.domain.name}", '-p', self.master.config.dirman_password]`